### ISSUE TITLE
More useful error handling

### ISSUE THREAD

**Issue by timstoop** [aelsabbahy/goss#82]
When trying to run my (obviously broken) script, I get the following error:

```
panic: regexp: Compile(`*\.* @@.*`): error parsing regexp: missing argument to repetition operator: `*`

goroutine 10 [running]:
panic(0x6e2940, 0xc82028ea80)
        /home/travis/.gimme/versions/go1.6.1.linux.amd64/src/runtime/panic.go:464 +0x3e6
regexp.MustCompile(0xc8200c7dc1, 0x9, 0x7d9648)
        /home/travis/.gimme/versions/go1.6.1.linux.amd64/src/regexp/regexp.go:232 +0x16f
github.com/aelsabbahy/goss/resource.newRegexPattern(0xc8200c7dc0, 0xb, 0x7d9a48)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/validate.go:169 +0x1e4
github.com/aelsabbahy/goss/resource.sliceToPatterns(0xc8200f33a0, 0x1, 0x1, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/validate.go:190 +0x428
github.com/aelsabbahy/goss/resource.ValidateContains(0x7fdf2e39adb8, 0xc820072f00, 0x7e4df0, 0x8, 0xc8200f33a0, 0x1, 0x1, 0xc82012be20, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/validate.go:228 +0x46f
github.com/aelsabbahy/goss/resource.(*File).Validate(0xc820072f00, 0xc820072640, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/file.go:55 +0xb7e
github.com/aelsabbahy/goss.Validate.func2(0xc820101080, 0xc8200eb020, 0xc8200eafc0, 0xc820072640)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/validate.go:69 +0xb3
created by github.com/aelsabbahy/goss.Validate
        /home/travis/gopath/src/github.com/aelsabbahy/goss/validate.go:72 +0x483
```

Although I recognise the error is somewhere in my code and not something in goss, but it would be helpful if it could point out a line number or something of the yaml.

**Comment by aelsabbahy** [comment]
Line number would be difficult, but I should be able to add something like #74 where it prints out the resource name and problematic attribute.

**Comment by timstoop** [comment]
That would work fine as well!

**Comment by aelsabbahy** [comment]
Once the above PR is released, the behavior will be as follows.

Given:
```yaml
file:
  /etc/passwd:
    exists: true
    contains: [/*\.* @@.*/]
```

Output:
```bash
$ goss validate
.F

Failures:

/etc/passwd: contains: Error: error parsing regexp: missing argument to repetition operator: `*`

Total Duration: 0.001s
Count: 2, Failed: 1

```

**Comment by aelsabbahy** [comment]
Fixed in [v0.1.7](https://github.com/aelsabbahy/goss/releases/tag/v0.1.7). Let me know if you run into any more issues.
